apps/test-suite/tests/SQLite.js

```diff
@@ -284,5 +284,150 @@ export function test(t) {
         });
       });
     }
+
+    if (Platform.OS !== 'web') {
```

Comment:
Oh, one more thing 😅 Shouldn't it work on web too?

Reply:
The first one should and does, in fact. The latter cannot, since web sql does not support PRAGMAs :(

I've enable first test for web.